tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/BaseTest.php

```diff
@@ -67,6 +68,10 @@ public function should_disable_tickets_and_rsvps_for_series(): void {
 	 * @test
 	 */
 	public function should_not_replace_tickets_block_on_post(): void {
+		if ( tec_using_classy_editor() ) {
```

Comment:
As per my previous comment, I think Classy should just be disabled in these tests, removing the need to skip tests if it's enabled.